sensor_msgs/PointCloud2[] clouds_to_classify
---
bool success
